src/app/auth/auth.service.ts

```diff
@@ -1,27 +1,29 @@
 import { Injectable, NgZone } from '@angular/core';
-import { User } from './models/user';
-// import  auth  from 'firebase/app';
+import { User } from './models/user.modal';
+import {patient} from './models/patient.model'
 import { AngularFireAuth } from '@angular/fire/auth';
 import {
   AngularFirestore,
   AngularFirestoreDocument,
 } from '@angular/fire/firestore';
 import { Router } from '@angular/router';
 import firebase from 'firebase';
-// import 'firebase/auth';
+import { Observable } from 'rxjs';
+
 
 @Injectable({
   providedIn: 'root',
 })
 export class AuthService {
   userData: any;
+  patients: Observable<any[]>;
 
   constructor(
     public afs: AngularFirestore,
     public afAuth: AngularFireAuth,
```

Comment:
`NgZone` is a cool thing ,but I'm not sure why is it required in this file,Please explain?